src/main/java/com/google/gcp_variant_transforms/library/SchemaGenerator.java

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.cloud.bigquery.Schema;
```

Comment:
Could you please confirm the followings:
 * How [`BigQueryIO.Write`](https://beam.apache.org/releases/javadoc/2.2.0/index.html?org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.html) accepts the schema?
* How can we convert `Schema` object to JSON or [`TableSchema`](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/com/google/api/services/bigquery/model/TableSchema.html?is-external=true)?